Automattic/newspack-newsletters
# Add block support for Site Logo, Site Tagline, and Site Title

### All Submissions:






### Changes proposed in this Pull Request:



Instead of using a hack to display the Site Logo or Site Title throughout the default layouts, this PR adds support for Core's Site Logo, Site Tagline, and Site Title.

Closes #646 .

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Make sure your site has a Site Logo
3. Create a new newsletter
4. You should see your logo in the preview
5. Add any default layouts
6. In the editor you should see the "Site Logo" block
7. Send a test email and check if everything is being displayed correctly
8. Go to the customiser and remove your Site Logo
9. Create a new newsletter
10. You should see your site title in the preview
11. Add any default layouts
12. In the editor you should see the "Site Title" block
13. Add the "Site Tagline" block
14. Send a test email and check if everything is being displayed correctly
15. Send and Publish the Newsletter
16. Check the web version

### Other information:
